.github/workflows/cmake.yml

```diff
@@ -98,22 +98,23 @@ jobs:
           apt-get update
           apt-get -y install libssl-dev:${TARGETARCH}
 
-      - name: install contemporary cmake
-        if: ${{ matrix.container != null }}
-        uses: lukka/get-cmake@latest
-
       - name: checkout workspace
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
-      - name: make build dir
-        run: cmake -E make_directory ./build
+      - name: install contemporary cmake
+        uses: lukka/get-cmake@latest
 
-      - name: configure cmake
-        env:
-          TOOLCHAIN: ${{ matrix.toolchain && format('./toolchains/{0}', matrix.toolchain) || '' }}
-        run: cmake ${{matrix.cmake_opts}} -DCMAKE_TOOLCHAIN_FILE="${TOOLCHAIN}" -S . -B ./build
+      - uses: lukka/run-vcpkg@v10
+        with:
+          vcpkgGitCommitId: '19af97cba8ca48474e4ad15a24ed50271a9ecdac'
```

Comment:
@ekoby Will you please add a note about this commit ID so we'll understand its significance and to inform future changes to the value?